ios/AppLovinMAX.m

```diff
@@ -923,6 +1045,11 @@ - (void)logInvalidAdFormat:(MAAdFormat *)adFormat
     [self log: @"invalid ad format: %@, from %@", adFormat, [NSThread callStackSymbols]];
 }
 
+- (void)logUninitializedAccessError:(NSString *)callingMethod
```

Comment:
Should we port this to other API calls as well?

Reply:
Ok, I'll update the Java side as well.